bot/bot.py

```diff
@@ -209,7 +212,7 @@ async def remove_order(ctx: commands.Context) -> None:
     brief="Outputs your current submitted orders.",
     description="Outputs your current submitted orders. "
     "In the future we will support outputting a sample moves map of your orders.",
-    aliases=["view", "vieworders"],
+    aliases=["v, view", "vieworders"],
```

Comment:
```suggestion
    aliases=["v", "view", "vieworders", "view-orders"],
```
v & view should be seperate, + view-orders I have heard